setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPLv2
+zip_safe=False
+entry_points={
```

Comment:
this is not correct for setup.cfg ..

you can find example in  doc, just take a look

http://docs.openstack.org/developer/pbr/#setup-cfg